frontend/src/services/index.js

```diff
@@ -8,7 +8,9 @@ if (localStorage.getItem("user")) {
 	API.interceptors.request.use(
 		(config) => {
 			let params = new URLSearchParams();
-			const { accessToken } = JSON.parse(localStorage.getItem("user"));
```

Comment:
Move this in another commit, because not link with the commit "Change rich text editor -- part One --"